### PR TITLE
Stake-Take gentx + peer

### PR DESCRIPTION
**Pull request by StakeTake**
- Stake-Take
- https://stake-take.com https://twitter.com/StakeAndTake
- Aleksandr7793#1495
- neutronvaloper1usukv06ykjkj4xgwpx6apu6xrksllex7pgx6u7

### CODE REVIEW

File: quark/gentxs/Stake-Take-gentx.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ity":"4A1DED53D477793B","website":"https://stake-take.com","security_contact":"team@stake-take.com","details":"Trustworthy and high performance validators. Stake with us and Take profit 🚀"},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1000000000000000000","delegator_address":"acre17gc07hawajnfg7e4539pmps0zfkwrdf4tg8t9m","validator_address":"acrevaloper17gc07hawajnfg7e4539pmps0zfkwrdf4jskm8x","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"ub30mcDbQp0pr2DpAt4K0VyS19uzFjuwN6++tZZUzZo="},"value":{"denom":"aacre","amount":"100000000000000000000"}}],"memo":"6b9d2a621be404e3beffecc0f434898d158ff4bc@138.201.198.240:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/ethermint.crypto.v1.ethsecp256k1.PubKey","key":"ArvY1PD5vmLw+Z4AuA59PHeYm1xqvts9HBhxjcsl9qom"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["x/Wcyd/z3SjHprkWVVQumjLYtPZ6FcQcLgCceUezVn4AWVZIhV8HhiFodyq6j1UAhHvm12cy4Yfd2+Vb38aZxQE="]}
```
Comment (albertandrejev) [inline quark/gentxs/Stake-Take-gentx.json:1]: Please set following properties in accordance with the documentation:
* `max_change_rate`
* `denom`
* `amount` of delegation

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/Stake-Take-gentx.json (hunk 1 above)
Comment (StakeTake) [reply quark/gentxs/Stake-Take-gentx.json]: Sorry mybad, gentx updated.

File: quark/gentxs/Stake-Take-gentx.json [hunk 2]
```diff
@@ -0,0 +1 @@
..."moniker":"Stake-Take","identity":"4A1DED53D477793B","website":"https://stake-take.com/","security_contact":"team@stake-take.com","details":" Trustworthy and high performance validators. Stake with us and Take profit 🚀 "},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1usukv06ykjkj4xgwpx6apu6xrksllex7m4lr66","validator_address":"neutronvaloper1usukv06ykjkj4xgwpx6apu6xrksllex7pgx6u7","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"9skwYh53S5fJ8YB0mOMNyAOlubD/Wmy7mKaXuK0TtnE="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"cb743a664a0e730974f3abdb88a1530407d44fa6@138.201.198.240:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AgE3tIDB1cV6JMN0zvbUd85IkUZElLLs73YQsbHls7Gx"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["N9mUh4AjJkEvCL7Sb4SZe0g5JsMLT0IvSpBnlxMoWTJQB0m/j6+f5JJpWnaDP8OKkWl3hlyxGXiXnzNEfaX1Qw=="]}
```
Comment (albertandrejev) [inline quark/gentxs/Stake-Take-gentx.json:1]: `max_change_rate` still contains wrong value.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/Stake-Take-gentx.json (hunk 2 above)
Comment (StakeTake) [reply quark/gentxs/Stake-Take-gentx.json]: created new gentx

**Pull request closed without merging**